src/web/routes/application/send-code/validate.js

```diff
@@ -2,7 +2,7 @@ const { check } = require('express-validator')
 const { translateValidationMessage } = require('../common/translate-validation-message')
 
 const validate = [
-  check('chooseChannelForCode').isIn(['text', 'email']).withMessage(translateValidationMessage('validation:selectTextOrEmailChooseChannelForCode'))
+  check('sendCode').isIn(['text', 'email']).withMessage(translateValidationMessage('validation:selectTextOrEmailSendCode'))
```

Comment:
Suggest that the field is called something like `channelForCode`

Reply:
Updated to channelForCode

Reply:
...or `methodForCode`.